cluster-state-service/handler/store/datastore.go

```diff
@@ -179,15 +212,20 @@ func resetStreamIdleTimer(t *time.Timer) {
 	t.Reset(streamIdleTimeout)
 }
 
-func handleGetResponse(resp *clientv3.GetResponse) map[string]string {
-	kv := make(map[string]string)
+func handleGetResponse(resp *clientv3.GetResponse) map[string]types.Entity {
+	kv := make(map[string]types.Entity)
 
 	if resp == nil || resp.Kvs == nil {
 		return kv
 	}
 
 	for _, response := range resp.Kvs {
-		kv[string(response.Key)] = string(response.Value)
+		entity := types.Entity{
+			Key: string(response.Key),
+			Value: string(response.Value),
+			Version: strconv.FormatInt(response.ModRevision, 10),
```

Comment:
Can you maybe add a comment or an etcd link that explains what the ModRevision is?